### PR TITLE
Keep activations in bidirectional LSTM (fixes: https://github.com/onnx/tensorflow-onnx/issues/2211)

### PR DESCRIPTION
**Pull request by MaximeChurin**
The forward and backward LSTMs have their correct activations but as soon as we rewrite it to a bidirectional this attribute is skipped and we get the default activations.

I am not sure how to write it in the `rewriter` file, you have something similar in the [bigru](https://github.com/onnx/tensorflow-onnx/blob/main/tf2onnx/rewriter/bigru_rewriter.py#L66) so let me know if you prefer the syntax with decode and comprehension list

### CODE REVIEW

**Comment by MaximeChurin** [comment]
@fatcat-z

**Comment by fatcat-z** [comment]
Thanks for your contributions.

Yes, please follow the way in bigru_rewriter.py to read activations from those LSTMs.

**Comment by MaximeChurin** [comment]
thanks@fatcat-z
It should be ok now, let me know

**Comment by fatcat-z** [comment]
> thanks@fatcat-z It should be ok now, let me know

The CI failures are caused by tf 1.15. For compatibility, we might need to check if they are None before we call decode() method.

**Comment by MaximeChurin** [comment]
good catch @fatcat-z , I made the change and the tests passed but now it differs a bit from the bigru implem. If you have any suggestions please make it via GitHub code review suggestion

File: tests/test_lstm.py [hunk 1]
```diff
@@ -775,6 +775,31 @@ def func(x):
             return tf.identity(y[0], name="output"), tf.identity(y[1], name="output1")
         self.run_test_case(func, {"input:0": x_val}, [], ["output:0", "output1:0"], rtol=1e-05, atol=1e-06)
 
+    @check_tf_min_version("2.0")
+    @skip_tf_versions("2.1", "Bug in TF 2.1")
```
Comment (fatcat-z) [inline tests/test_lstm.py:779]: Which bug are we talking about here? Could you please put the url into the comment?

Comment (MaximeChurin) [reply tests/test_lstm.py:779]: no clue I copied the structure of the bigru test that had this decorator, do you think I can remove it?

Comment (fatcat-z) [reply tests/test_lstm.py:779]: > no clue I copied the structure of the bigru test that had this decorator, do you think I can remove it?

Yes please, let's have a try.

Comment (fatcat-z) [reply tests/test_lstm.py:779]: BTW, TF 2.1 has been removed out of the CI pipeline, so this won't be run anyway.

Comment (MaximeChurin) [reply tests/test_lstm.py]: done, waiting for the CI to complete

**Review by fatcat-z — APPROVED** [review]
LGTM, thanks for your contributions!

**Pull request merged**